src/useSubtitles.js

```diff
@@ -0,0 +1,114 @@
+import { getVoiceFromCue, isJSON, toIterable } from './machinery/utilities';
+import { initial, initialCurrentSubtitle } from './defaults';
+import { useCallbackRef } from "./machinery/useCallbackRef";
+import { useEvent } from "./machinery/useEvent";
+
+export function useSubtitles({ language = "nl" }) {
```

Comment:
Ik zou het meegeven van een taal verplicht maken, dat is denk ik logischer dan nederlands als default.
```suggestion
export function useSubtitles({ language }) {
```